WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -10,7 +10,9 @@ def index
           persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
         end
         if params[:search].present?
-          persons = persons.where("rails_persons.name LIKE :input OR wca_id LIKE :input", input: "%#{params[:search]}%")
+          params[:search].split.each do |part|
+            persons = persons.where("rails_persons.name LIKE :input OR wca_id LIKE :input", input: "%#{part}%")
```

Comment:
In fact, I think we could call `Users.search(params[:search], persons_table: true)` here instead?